cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/UpscaleStackHandler.java

```diff
@@ -60,10 +63,14 @@ public void accept(Event<UpscaleStackRequest> upscaleStackRequestEvent) {
             if (!task.completed(statePollerResult)) {
                 statePollerResult = syncPollingScheduler.schedule(task);
             }
```

Comment:
For me this form is more readeable https://github.com/sequenceiq/cloudbreak/commit/bf8562e33b121844816428fc71cdf9bc6b0d21d9#diff-9b287884a2c2878ba2688f98db6ee7e0R31